### DISCUSSION TITLE
Achievements for teachers?

### DISCUSSION THREAD

**Discussion by manuelabarca** [community/community#18187] [Other Feature Feedback, Questions, & Ideas]
Now that the Achievements have come out, one would be interesting for those who are teachers or students of having a good time hehe

**Comment by Synceratus** [comment]
Would be nice to have them, pretty sure that they're going to make more soon as the community likes them.

**Comment by ettaboyle** [comment]
Hi @manuelabarca Thanks for this comment! To be sure the profile product team sees this I've added the "GitHub Education" & "enhancement" labels and moved this to the "profile" category
